_templates/community.html

```diff
@@ -57,7 +57,7 @@ <h2 id="irc"><img src="/img/icons/ico_help.svg" class="titleicon" alt="Icon">{%
 </div>
 </div>
 
-<h2 id="nonprofit"><a name="{% translate non-profit anchor.community %}">{% translate nonprofit %}</a></h2>
+<h2 id="nonprofit-organizations"><a name="{% translate non-profit anchor.community %}">{% translate nonprofit %}</a></h2>
```

Comment:
You're correct that we're linking to the anchor "nonprofit", but that's because we're linking to the translated `<a name` anchor next to it.

I mention this in the commit notes, that the problem is that the anchor is used twice on the page (the `id=nonprofit` was added just a couple weeks ago here: https://github.com/bitcoin/bitcoin.org/pull/813/files ).

Also, note that HTML Proofer checks all of our internal links to anchors, so if I had broken an anchor, it would've told us.  :-)